app/containers/Wallet.js

```diff
@@ -41,12 +40,18 @@ class Wallet extends React.Component {
     }, 60000);
     // Get peer info every 10 seconds, so we can no if there are no available
     // peers.
-    this.props.setInterval(() => {
-      if (politeiaEnabled) {
-        getPeerInfo();
-      }
+    this.peerInfoIntervalRef = this.props.setInterval(() => {
+      getPeerInfo();
     }, 10000);
   }
+  componentWillUnmount() {
```

Comment:
I already did this in https://github.com/decred/decrediton/pull/2901 after refactoring this container to be
functional component, I would recommend merging #2901 then merge this one for the other part which isn't covered in #2901

Reply:
What do you think guys @vctt94 @alexlyp ?
Would it make sense rebasing 2901 and merge it at this level ? 
Looks like the intervals cleanup is essential performance improvement.

Reply:
I think that's fine, can you rebase it, so I can review it?

Reply:
Surething, on it now